### ISSUE TITLE
Playing non-exsting buildable animations distorts the model.

### ISSUE THREAD

**Issue by Viech** [Unvanquished/Unvanquished#521]
If a buildable animation is triggered and the animation in question was never registered, instead of printing a warning/error message, a bad animation will be played, distorting the model.

**Comment by illwieckz** [comment]
Instruction is needed to reproduce the issue.

**Comment by bmorel** [comment]
Then it's more like a task/cleanup, than a bug. Also, is this still true? I second illwieckz that instructions on how to check if this is true or false would be really appreciated, especially if a desync has to be artificially made.

**Comment by VReaperV** [comment]
I don't think this is the case anymore, if the requested animation is not loaded it will use the default obe, resulting in identity skeleton.
